### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by vinh-tranhuu**

### CODE REVIEW

**Review by vpkonkiel — CHANGES_REQUESTED** [review]
Lesly added

File: about-us.txt [hunk 1]
```diff
@@ -1,24 +1,14 @@
...
-
 I'm Jonny :)
-
 sam...
-
-
+Im Les
 sam???
-
+kev again
```
Comment (Alsm867) [inline about-us.txt:6]: approved

**Review by vpkonkiel — APPROVED** [review]
good to go

File: about-us.txt [hunk 2]
```diff
@@ -1,24 +1,14 @@
...
-
-Tyler g
-
-
 I'm Jonny :)
-
 sam...
-
-
+Im Les
```
Comment (Alsm867) [inline about-us.txt:4]: I'll allow it

File: about-us.txt [hunk 3]
```diff
@@ -1,24 +1,14 @@
...
 Tyler g
-
 Richard
-
-
 Cory
 sam a third time
 sam a fourth time muahaha
 sam with a branch merge
+
```
Comment (Alsm867) [inline about-us.txt:14]: hahahaha

**Review by Alsm867 — APPROVED** [review]

**Pull request merged**